client/src/app/pages/migration-waves/components/ticket-issue.tsx

```diff
@@ -1,25 +1,26 @@
 import React from "react";
-import { Text } from "@patternfly/react-core";
+import { Text, TextVariants } from "@patternfly/react-core";
 
 import { Ticket } from "@app/api/models";
-import { useTrackerTypesByProjectId } from "@app/queries/trackers";
+import { useTranslation } from "react-i18next";
+import ExternalLink from "@app/components/ExternalLink";
 
 export interface ITicketIssueProps {
   ticket?: Ticket;
 }
 
 export const TicketIssue: React.FC<ITicketIssueProps> = ({ ticket }) => {
-  const useTicketIssue = () => {
-    const types = useTrackerTypesByProjectId(
-      ticket?.tracker?.name,
-      ticket?.parent
-    );
-    const type = types.find((kind) => kind.id === ticket?.kind);
-    if (type) return type.name;
-    return "";
-  };
+  const { t } = useTranslation();
 
-  const ticketIssue = useTicketIssue();
-
-  return <Text>{ticketIssue}</Text>;
+  return (
+    <>
```

Comment:
The extra `<></>` wrapper isn't needed.

Looks like previously the column text would be the name of the type of the issue tracker -- so maybe something like "Jira1" or whatever the instance name of a tracker would be.  Probably a good idea to keep that as the URL/Link could be long like "https://issues.redhat.com/browse/MTA-701" vs "Red Hat Jira".